packages/venia-concept/package.json

```diff
@@ -52,6 +52,7 @@
     "@babel/runtime": "~7.4.2",
     "@magento/babel-preset-peregrine": "~1.1.0",
     "@magento/eslint-config": "~1.5.0",
+    "@magento/i18n-defaults": "~0.0.1",
```

Comment:
We will have to publish the extension to npm before merging this PR.

Reply:
Why would we want to provide every install with a French language pack from an example extension? I thought this language pack was to enable testing when required?

Reply:
We do need a less manual way to provide it for the production deployments of `master` and `develop` as we have added a french store view to the backends to demonstrate the functionality. I believe @dpatil-magento has uploaded a copy of the `fr_FR.json` file to the CI system somewhere but I expect he has to manually update it.

On top of that, I had assumed we would just not include this dependency when scaffolding, but I realize that not everyone uses scaffolding (some just clone/fork) so that's probably not a valid assumption.

Sounds like we need to come up with a way optionally include this dependency, but as you are supposed to include extensions as a dependency of your project, I'm not really sure how we can do that here. Maybe in the extension we can add an environment variable to only include it when necessary?

Reply:
Silly me, moved this to an opt-in experience. @dpatil-magento can set up the CI to install it. Developers can add it as a dependency (or link it) locally. Just need to make sure we on the core team don't accidentally commit it to venia-concept's package.json!